requirements/base.txt

```diff
@@ -1,4 +1,5 @@
 # edX Internal Requirements
+edx-i18n-tools
```

Comment:
You probably want to ensure that the version used is >= 0.4.0, which introduced logic to exit from `validate` with a non-zero code when problems are found.

Reply:
if we dnt specify version it automatically installs latest.

Reply:
Does it do that even when it has a copy of an older version already?

Reply:
@amangano-edx Im not sure how travis handles that. But before that it was using 0.3.9 now its using 0.4.3 which can be seen here https://travis-ci.org/edx/edx-ora2/jobs/362037159